.github/workflows/on_run_create_jira_flaky_test.yml

```diff
@@ -0,0 +1,66 @@
+name: On Run Create Jira for Flaky Test
```

Comment:
I'll leave this up to you, but IMO we can make the action names and file names less verbose, e.g:

File: flaky_test_reporter.yml
Name: Flaky Test Reporter

File: infinispan_tests.yml
Name: Infinispan Tests

Reply:
I read somewhere that naming like <trigger>-<verbalized action> helps to keeps things understandable, let's see if it's true